bin/convert-argv.js

```diff
@@ -3,6 +3,10 @@ var fs = require("fs");
 fs.existsSync = fs.existsSync || path.existsSync;
 var interpret = require("interpret");
 var prepareOptions = require("./prepareOptions");
+var webpackConfigurationSchema = require("../schemas/webpackConfigurationSchema.json");
+var validateSchema = require("webpack/lib/validateSchema");
```

Comment:
Please use `webpack.validateSchema ` instead

Reply:
Can you switch this over @jbottigliero?